src/renderer/runner.ts

```diff
@@ -350,59 +374,57 @@ export class Runner {
   }
 
   /**
-   * Execute Electron.
-   *
-   * @param {string} dir
-   * @returns {Promise<RunResult>}
-   * @memberof Runner
+   * Actually runs the fiddle with either local electron build
+   * or the user selected electron version
    */
-  public async execute(dir: string): Promise<RunResult> {
-    const {
-      currentElectronVersion,
-      flushOutput,
-      pushOutput,
-      executionFlags,
-    } = this.appState;
-
-    const { version, localPath } = currentElectronVersion;
-    const binaryPath = getElectronBinaryPath(version, localPath);
-    console.log(`Runner: Binary ${binaryPath} ready, launching`);
+  private async runFiddle(params: RunFiddleParams) {
```

Comment:
While fixing tests -- realized that this function always returns `RunResult.INVALID`
Fixed this now :)